src/app/valid-email-message/valid-email-message.component.html

```diff
@@ -0,0 +1,11 @@
+ <p class="header-message">Request Sent!</p>
+ <br><br>
+ <p>
+   Hang tight! You will receive an email invitation
```

Comment:
I love the message, put it in the localization files.

Reply:
Done